pulsar-broker/src/test/java/org/apache/pulsar/client/impl/MessageChunkingTest.java

```diff
@@ -395,6 +427,7 @@ public void testResendChunkMessages() throws Exception {
         receivedMsg = consumer.receive(5, TimeUnit.SECONDS);
         assertEquals(receivedMsg.getValue(), "chunk-1-0|chunk-1-1|chunk-1-2|");
         consumer.acknowledge(receivedMsg);
+        Assert.assertEquals(((ConsumerImpl<String>) consumer).getAvailablePermits(), 10);
```

Comment:
Why should the available permits be 10?

Reply:
There are 8 chunks processed and receive messages twice.